spec/ruby/core/class/name_spec.rb

```diff
@@ -0,0 +1,12 @@
+require_relative '../../spec_helper'
+require_relative 'fixtures/classes'
+
+describe "Class#name" do
```

Comment:
Sorry to complain about spec location, but this should be in `spec/ruby/core/module/name_spec.rb`.

Maybe it would be useful to have something like `mspec where "Class.new.method(:name)"` or `mspec where Class#name` that would print `spec/ruby/core/module/name_spec.rb`?

Reply:
Sorry I don't know what you mean about the `mspec...` method call there?

Reply:
It was just an idea for more easily finding out in which file the specs of a method should go.

Reply:
Moved.